.github/workflows/freebsd.yml

```diff
@@ -24,15 +24,15 @@ jobs:
       with:
         path: libva-utils
     - name: test
-      uses: vmactions/freebsd-vm@v0.1.3
+      uses: vmactions/freebsd-vm@v0.1.5
       with:
         prepare: |
           pkg install -y meson pkgconf libdrm libXext libXfixes wayland
           pkg install -y -x '^mesa($|-libs)'
         run: |
           set PREFIX="$PWD/_install"
           set LIBDIR="$PREFIX/lib"
-          setenv PKG_CONFIG_PATH $LIBDIR/pkgconfig
+          setenv PKG_CONFIG_PATH $LIBDIR/pkgconfig:$PREFIX/libdata/pkgconfig
```

Comment:
@jbeich : initially build was failing. I see that libva.pc is not installed in $PREFIX/libdata/pkgconfig instead of $LIBDIR/pkgconfig as it was before. Is that a chance on bsd side?

Reply:
libdata/pkgconfig is [set downstream](https://github.com/freebsd/freebsd-ports/blob/6f921bef373f/devel/pkgconf/Makefile#L23) but libva installs `*.pc` files manually instead of relying `import('pkgconfig').generate()` which [supports downstream default](https://github.com/mesonbuild/meson/commit/aba8792b6657).

Reply:
[Originally](https://github.com/freebsd/freebsd-ports/commit/4edf9758c2c8), `libdata/pkgconfig` was set due to [hier(7)](https://man.freebsd.org/hier/7) defining `lib/` as the place for actual libraries. However, after `lib/` gained `dtrace/` and `debug/` the distinction with `libdata/` became moot. Maybe @bapt knows about plans to resolve this mess. For example, [relaxing the default](https://github.com/freebsd/freebsd-ports/commit/06da414d5fe9) was considered 15 years ago.

Reply:
> but libva installs *.pc files manually instead of relying on import('pkgconfig').generate()

Ok, this has changed. We got contribution which has removed manual stuff and switched to `import('pkgconfig')`. See this commit: https://github.com/intel/libva/commit/edf62869efb25efa68f4aaea9627f7bfc8ffbfdb

@jbeich : what would be your recommendation for ci job we have here in libva-utils? Should I just use libdata/pkgconfig and remove lib/pkgconfig or it's better to have both paths till your folks will decide what should be the default on freebsd?

Reply:
> Should I just use libdata/pkgconfig and remove lib/pkgconfig

Doesn't matter much. If you plan to install more dependencies via CMake, autotools, etc. then `lib/pkgconfig` would still be required. Older build systems don't have a well-defined rules to **generate** and **install** `*.pc` files, so `lib/pkgconfig` is often hardcoded in each project. Downstream can still influence where such build systems **search** `*.pc` files, so everything is often fine until one tries to install dependencies outside of ports/packages.

Reply:
Ok, so, I'll leave it as it is now.